### PR TITLE
feat(LICENSE): Relicenses to Apache 2 with LLVM exception

### PR DESCRIPTION
**Pull request by thomastaylor312**
This is in preparation for moving this repo into the Bytecode alliance as indicated in https://github.com/bytecodealliance/governance/issues/88

### CODE REVIEW

**Comment by thomastaylor312** [comment]
@calvinrp since you also have a commit to this repo, I'd like to confirm that you're ok with me relicensing those contributions to Apache 2 with LLVM exception

**Review by calvinrp — APPROVED** [review]

**Comment by calvinrp** [comment]
> @calvinrp since you also have a commit to this repo, I'd like to confirm that you're ok with me relicensing those contributions to Apache 2 with LLVM exception

Ok with me.

**Comment by jsturtevant** [comment]
LGTM

**Review by jsturtevant — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 
+   Copyright 2023, Bytecode Alliance
```
Comment (jsturtevant) [inline LICENSE:189]: this could be updated to 2024?

Comment (thomastaylor312) [reply LICENSE]: Done!

**Pull request merged**